### PR TITLE
Update Dockerfile

### PR DESCRIPTION
**Pull request by Jeremy-Gaillard**
Updated Dockerfile and added run and build instructions for it.

I had to change the openjdk image that is now deprecated. I replaced it by  the eclipse-temurin one which seems to be the closest successor.

### CODE REVIEW

File: Usage.md [hunk 1]
```diff
@@ -16,6 +16,22 @@ The app will start with subscriptions defined in `resources/subscription-example
...
+
+```shell
+docker build . -t anshar
+```
+
+Run the image :
+
+```shell
+# With default coniguration
+docker run -p 8012:8012
```
Comment (lassetyr) [inline Usage.md:31]: The docker-commands are incomplete:
```
>  docker run -p 8012:8012                                                                                                                                                                                                       
"docker run" requires at least 1 argument.
See 'docker run --help'.
```

**Review by lassetyr — CHANGES_REQUESTED** [review]

File: Usage.md (hunk 1 above)
Comment (Jeremy-Gaillard) [reply Usage.md]: Oops, I forgot to put the container name. Fixed it.

**Comment by Jeremy-Gaillard** [comment]
Note: for some reason, I am not able to stop the container using Ctrl+C. I always have to kill the container. This was already the case with the old version of Docker. I'm not knowledgeable enough in Java to understand where this comes from.

**Comment by lassetyr** [comment]
> Note: for some reason, I am not able to stop the container using Ctrl+C. I always have to kill the container. This was already the case with the old version of Docker. I'm not knowledgeable enough in Java to understand where this comes from.

FYI: I fixed this in commit https://github.com/entur/anshar/commit/a27d6b1bd2774468f873bf6e02407184e7f395bc

**Pull request merged**